style_editor.html

```diff
@@ -10203,6 +10203,11 @@ <h4 class="settings-section-label" title="These settings apply to the whole page
               <span>Dark Mode</span>
             </label>
 
+            <label title="Toggles these pop-up tooltips On or Off.">
```

Comment:
I think this is weird phrasing.
Maybe remove "these" ?
Or maybe something like "Toggles tooltips (like the one you are reading right now) On or Off." ?

Reply:
Sure how about "Toggles pop-up tooltips such as this On or Off." ?

Reply:
updated.

Reply:
I think "this" or "these" is a little too meta, and it will take people a few seconds to connect the dots to understand what that means I think.

We could just make it "Toggles pop-up tooltips On or Off".

Reply:
Done